### PR TITLE
[Admin] Add update product layout, get product info, show product

### PR DESCRIPTION
**Pull request by at-khadinh**
## Description
* Add update product layout.
* Get and show product information
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

```
mysql> explain select * from `categories`\G
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: categories
   partitions: NULL
         type: ALL
possible_keys: NULL
          key: NULL
      key_len: NULL
          ref: NULL
         rows: 15
     filtered: 100.00
        Extra: NULL
1 row in set, 1 warning (0.00 sec)
```
```
mysql> explain select * from `products` where `products`.`id` = '1' and `products`.`deleted_at` is null limit 1\G
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: products
   partitions: NULL
         type: const
possible_keys: PRIMARY
          key: PRIMARY
      key_len: 4
          ref: const
         rows: 1
     filtered: 100.00
        Extra: NULL
1 row in set, 1 warning (0.01 sec)
```

### CODE REVIEW

File: resources/views/admin/pages/products/edit.blade.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        <div class="x_title">
+          <h2>@lang('product.update.table-title')</h2>
+          <div class="clearfix"></div>
+        </div>
+        <div class="x_content">
+          <br />
+          <form data-parsley-validate method="POST" action="{!! route('admin.products.update', ['id' => $product['id']]) !!}" enctype="multipart/form-data" class="form-horizontal form-label-left">
+
+            @csrf
+
```
Comment (at-xinhnguyen) [inline resources/views/admin/pages/products/edit.blade.php]: chỗ này thiếu `@method('PUT')`  em

File: resources/views/admin/pages/products/edit.blade.php [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            <div class="form-group">
+              <label class="control-label col-md-3 col-sm-3 col-xs-12">@lang('product.create.category')</label>
+              <div class="col-md-6 col-sm-6 col-xs-12">
+                <select name="category_id" class="select2_single form-control" tabindex="-1">
+                  @foreach ( $categories as $category )
+                    @if ($product->category_id == $category->id)
+                      <option value="{{ $category->id }}" selected="selected">{{ $category->name }}</option>
+                    @else
+                      <option value="{{ $category->id }}">{{ $category->name }}</option>
```
Comment (at-xinhnguyen) [inline resources/views/admin/pages/products/edit.blade.php]: chỗ này khỏi cần if else. chỉ cần như này cho gọn em
<option value="{{ $category->id }}" {{ $product->category_id == $category->id ? 'selected' : '' }}>
    {{ $category->name }}
</option>

**Review by ghost — CHANGES_REQUESTED** [review]
Please check my cmt

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**